pallets/validators-manager/src/confirmations.rs

```diff
@@ -0,0 +1,60 @@
+use frame_support::log;
+use pallet_ethereum_transactions::ethereum_transaction::{
+    ActivateCollatorData, DeregisterCollatorData,
+};
+use sp_io::hashing::keccak_256;
+
+const PACKED_KEYS_SIZE: usize = 96;
+
+#[cfg(test)]
```

Comment:
Do we want this here?

Reply:
Move it to lib.rs with the rest of the tests

Reply:
Why? The context is much better this way. This file has this set of tests for these functions.

Reply:
Having these paths scattered all over the place will make it harder to see where they are. What is the problem of having them in 1 place, grouped together? What is the rational to change this existing practise?

Reply:
The rationale behind this proposal is to improve modularity within our codebase. Currently, the lib.rs file is responsible for organizing modules and tests. Instead, a simpler approach would be for each module to specify the path to its test files as a self-contained unit. It is a more modular approach I think.